main.py

```diff
@@ -0,0 +1,31 @@
+import streamlit as st
+
+from main_summarizer import Summarizer
+from syntax_summarizer import get_text
+
+if __name__ == '__main__':
+	cnn_summarizer = Summarizer()
```

Comment:
This is still global code, you just put it under the `if` statement  🤷‍♂️
In this way, code is less reusable